src/contrib/getpot/getpot.h

```diff
@@ -532,7 +532,7 @@ class GetPot
    * multiple threads at once, so we'll wrap access to
    * mutable objects in a mutex.
    */
-  GETPOT_MUTEX_DECLARE;
+  GETPOT_MUTEX_DECLARE
 
```

Comment:
When I was compiling I didn't have libmesh, so that macro was empty.  `-Wpedantic` complains when you have a single stray semicolon, so I put the semicolon inside the macro and left it off here.